pkg/sensors/tracing/generickprobe.go

```diff
@@ -355,6 +355,9 @@ func createMultiKprobeSensor(sensorPath string, multiIDs []idtable.EntryID) ([]*
 		maps = append(maps, socktrack)
 	}
 
+	killerDataMap := program.MapBuilderPin("killer_data", "killer_data", load)
```

Comment:
Let's use https://github.com/cilium/tetragon/blob/506677dc9b70f7d87fe23ac4cfcb2e45481e0ee1/pkg/sensors/tracing/killer.go#L24?

Reply:
nice, did not see that, thanks